plugins/poll/assets/javascripts/widgets/discourse-poll.js.es6

```diff
@@ -685,6 +699,32 @@ export default createWidget("discourse-poll", {
     this.state.showResults = !this.state.showResults;
   },
 
+  exportResults() {
+    const { attrs } = this;
+
+    // This uses the Data Explorer plugin export as CSV route
+    // There is detection to check if the plugin is enabled before showing the button
+    ajax("/admin/plugins/explorer/queries/-16/run.csv", {
+      type: "POST",
+      data: {
+        params: JSON.stringify({ // needed for data-explorer route compatibility
+          poll_name: attrs.poll.get("name"),
+          post_id: attrs.post.id.toString() // needed for data-explorer route compatibility
+        }),
+        explain: false,
+        limit: 1000000,
+        download: 1
+      }
+    }).then(( csvContent ) => {
+      const pom = document.createElement('a');
```

Comment:
what does pom means here ?

Reply:
renamed, that was just lazy copy & paste :stuck_out_tongue: